File: Katas/JavaScript/7KYU/Sum of Triangular Numbers.js
function sumTriangularNumbers(n) {

    if (n <= 0)
        return 0;

    let numbersArray = [];
    let currentNumber = 0;

    for (let i = 1; i <= n; i++) {
        currentNumber = i + currentNumber;
        numbersArray.push(currentNumber);
    }
    return numbersArray.reduce(function (a, b) { return a + b }, 0);
}

File: Katas/C#/7KYU/Sum of Triangular Numbers.cs
public class Kata
{
    public static int SumTriangularNumbers(int n)
    {
        int sum = 0;
        for (int i = 0, j = 1; i < n; i++, j += i + 1)
        {
            sum += j;
        }
        return sum;
    }
}